.github/workflows/CI.yml

```
@@ -57,7 +57,7 @@ jobs:
             Pkg.instantiate()'
       - run: julia --project=perf perf/samplers.jl
       - uses: julia-actions/julia-processcoverage@v1
-      - uses: codecov/codecov-action@v3
+      - uses: codecov/codecov-action@v4
         with:
```

v4 of the action does not support tokenless updates anymore. So the codecov token has to be added as a Github secret by an admin and the one can use something like
```suggestion
        with:
          token: ${{ secrets.CODECOV_TOKEN }} # required
          fail_ci_if_error: true
```